explorer/lib/explorer_web/endpoint.ex

```diff
@@ -33,9 +33,9 @@ defmodule ExplorerWeb.Endpoint do
     plug Phoenix.CodeReloader
   end
 
-  plug Phoenix.LiveDashboard.RequestLogger,
-    param_key: "request_logger",
-    cookie_key: "request_logger"
+  # plug Phoenix.LiveDashboard.RequestLogger,
+  #   param_key: "request_logger",
+  #   cookie_key: "request_logger"
```

Comment:
Commented code ?

Reply:
Commented elixir dashboard to disable it 👍